CMakeLists.txt

```diff
@@ -25,10 +25,20 @@
 # 5. And release mode (/m[:<N>] is also supported)
 #        msbuild rocksdb.sln /p:Configuration=Release
 #
+# To create JNI project add -DJNI=1 option
+#
 
 cmake_minimum_required(VERSION 2.6)
 project(rocksdb)
 
+set(USE_JNI_DEFAULT 0)
```

Comment:
But JNI is not third party library. The option there is just to instruct cmake to generate JNI project.  If you really do not like it, I can move it there.